traitlets/traitlets.py

```diff
@@ -801,6 +800,12 @@ def class_own_traits(cls, **metadata):
         return {n: t for (n, t) in cls.class_traits(**metadata).items()
                 if getattr(sup, n, None) is not t}
 
+    def has_trait(self, name):
+        """Returns True if the specified name is the name of a trait
+        attribute."""
```

Comment:
The summary in a docstring should generally be kept to one line. Something like "Returns True if the object has a trait with the specified name."